editor.planx.uk/src/ui/editor/SimpleMenu.tsx

```diff
@@ -14,7 +15,11 @@ interface Props {
   }>;
 }
 
-export default function SimpleMenu({ items, ...restProps }: Props): FCReturn {
+export default function SimpleMenu({
+  items,
+  children,
+  ...restProps
+}: Props): FCReturn {
```

Comment:
```suggestion
export default function SimpleMenu({
  items,
  children,
  ...restProps
}: PropsWithChildren<Props>): FCReturn {
```

React provides us with a nice utility type here that we can lean on 👌

Reply:
Noted for future addition of children to components 👍